packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -169,13 +169,30 @@ export class FlowBuilderApi {
     locale: string
   ): boolean {
     const result = node.content.keywords.find(
-      i => i.locale === locale && this.containsAnyKeywords(input, i.values)
+      keywords =>
+        keywords.locale === locale &&
+        this.containsAnyKeywords(input, keywords.values)
```

Comment:
suggestion: I would rename it to `inputMatchesAnyKeyword()`, I think it identifies better what the function is doing.